src/main/java/connectingstar/tars/habit/controller/HabitController.java

```diff
@@ -22,6 +24,7 @@
 public class HabitController {
 
     private final RunHabitCommandService runHabitCommandService;
+    private final QuitHabitQueryService quitHabitQueryService;
 
     @PostMapping(value = "/")
```

Comment:
```suggestion
    @PostMapping
```
path가 없다면 value는 생략해주세용

Reply:
알겠습니다!